README.md

```diff
@@ -1,2 +1,74 @@
 # apple-main-webpage-
```

Comment:
The project title has already been re-defined in line 6 below. To avoid repetition, kindly remove it.